shell/platform/android/io/flutter/embedding/android/AndroidTouchProcessor.java

```diff
@@ -80,8 +80,8 @@ public class AndroidTouchProcessor {
     int UNKNOWN = 4;
   }
 
-  // Must match the unpacking code in hooks.dart.
-  private static final int POINTER_DATA_FIELD_COUNT = 35;
+  // This value must match kPointerDataFieldCount in pointer_data.cc.
```

Comment:
Should this mention `AndroidTouchProcessor.java`?

Reply:
My idea is to have a centralized place to record the mirrored values instead of having all values to mention each other. So if you're to modify this value in `AndroidTouchProcessor`, you would realized "ok I have to change pointer_data.cc as well", then you go to `pointer_data.cc` and find other values to change accordingly.

Reply:
Gotcha. If we go with this approach, I would document this so that this pattern is clear:

1. I would update `pointer_data.cc` to declare itself as the centralized location that should be kept up-to-date with all other locations
2. I would update files that point to all other locations (like `pointer_data.h`) to point to `pointer_data.cc`
3. For comments that point to `pointer_data.cc`, I would add something like "See pointer_data.cc for the centralized list of locations that must be kept consistent".

Reply:
Done. Can you take a look to see if the words are clear enough?